ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -307,3 +307,15 @@
 # Marker for skipping tests if the cluster doesn't have configured cluster-wide
 # encryption with KMS properly
 skipif_no_kms = pytest.mark.skipif_no_kms
+
+# Squad marks
+red_squad = pytest.mark.red_squad
+brown_squad = pytest.mark.brown_squad
+green_squad = pytest.mark.green_squad
+blue_squad = pytest.mark.blue_squad
+red_squad = pytest.mark.red_squad
```

Comment:
We have `red_squad` listed twice
```suggestion
```

Reply:
Good catch! Sorted the definition alphabetically and removed the duplicate.